package.json

```diff
@@ -129,12 +129,12 @@
     "ovh-angular-tail-logs": "ovh-ux/ovh-angular-tail-logs#^1.1.2",
     "ovh-angular-toaster": "ovh-ux/ovh-angular-toaster#^0.8.0",
     "ovh-angular-user-pref": "ovh-ux/ovh-angular-user-pref#^0.3.1",
-    "ovh-api-services": "ovh-ux/ovh-api-services#^3.16.2",
+    "ovh-api-services": "ovh-ux/ovh-api-services#^3.18.0",
     "ovh-common-style": "ovh-ux/ovh-common-style#^3.2.2",
     "ovh-jquery-ui-draggable-ng": "ovh-ux/ovh-jquery-ui-draggable-ng#^0.0.5",
     "ovh-manager-webfont": "ovh-ux/ovh-manager-webfont#^1.0.1",
-    "ovh-ui-angular": "~2.17.1",
-    "ovh-ui-kit": "~2.17.1",
+    "ovh-ui-angular": "2.20.0",
```

Comment:
Axel said no :P

Reply:
That shouldn't be decided by any library.

`yarn.lock` files are here to prevent any `sideEffects`.

We explicitly have to run `yarn upgrade foo` or `yarn upgrade-interactive` commands and decide which dependencies must be upgraded.